cluster-provision/gocli/docker/docker.go

```diff
@@ -207,6 +207,8 @@ func NewCleanupHandler(cli *client.Client, cleanupChan chan error, errWriter io.
 	go func() {
 		createdContainers := []string{}
 		createdVolumes := []string{}
+		containerRemoveRetry := []string{}
```

Comment:
I find this variable name a bit odd to read at first. Maybe we can rename this to reflect better what it holds? Suggestion:
```suggestion
		containersFailedToRemove := []string{}
```

Reply:
Yes I think your suggested name is better - I will make the update - thanks @dhiller !